src/fabric-video/index.scss

```diff
@@ -2,6 +2,7 @@
 
 .creative--fabric-video {
     margin: auto;
+    max-width: 1920px;
```

Comment:
Yeah I was actually surprised this limit only applied to Fabric Video. I might use it elsewhere, so will `var` the hell out of it 😆